src/main/java/nl/tudelft/pixelperfect/event/EventParameter.java

```diff
@@ -0,0 +1,72 @@
+package nl.tudelft.pixelperfect.event;
+
+/**
+ * Class representing a single parameter of an Event, having either a discrete value
+ * (EventParameterValue) or a continuous one (Integer).
+ * 
+ * @author Jesse Tilro
+ *
+ */
+public class EventParameter {
+
+  private String key;
+  private EventParameterValues value;
+  private int number;
```

Comment:
Why make attribute names so complicated? You never use them when they are well encapsulated (like here), you are only using the class's public interface. In addition there is a lot of javadoc surrounding the attribute and the methods it it involved in, so regardless of the name it should be perfectly clear what it stands for. Nonetheless I will take your comments into account and refactor it to something like number value, thanks.